apps/api/v2/src/ee/event-types/event-types_2024_06_14/services/event-types.service.ts

```diff
@@ -35,22 +26,13 @@ export class EventTypesService_2024_06_14 {
     private readonly schedulesRepository: SchedulesRepository_2024_06_11
   ) {}
 
-  async createUserEventType(user: UserWithProfile, body: CreateEventTypeInput_2024_06_14) {
+  async createUserEventType(user: UserWithProfile, body: InputEventTransformed_2024_06_14) {
     await this.checkCanCreateEventType(user.id, body);
     const eventTypeUser = await this.getUserToCreateEvent(user);
-    const {
-      bookingLimits,
-      durationLimits,
-      periodType = undefined,
-      periodDays = undefined,
-      periodCountCalendarDays = undefined,
-      periodStartDate = undefined,
-      periodEndDate = undefined,
-      recurrence = undefined,
-      ...bodyTransformed
-    } = this.inputEventTypesService.transformInputCreateEventType(body);
+    const { destinationCalendar: _destinationCalendar, ...rest } = body;
```

Comment:
Can destination calendar be omitted from `InputEventTransformed_2024_06_14` so that we don't take it out and simply don't use? What is its use if we destructure here and do nothing with it?

Reply:
we don't want to pass `destinationCalendar` to `createEventType` but pass to `updateEventType`